### PR TITLE
Release notes

### PR DESCRIPTION
**Pull request by SamTheisens**
Adds release notes to the documentation.
Prepares us for automated `.yml` upgrades, because we'll need to know all historical releases in the right order.

*Testing:*
No testing is needed, because there are no changes to library code.

----
### 📕 [TECH-470](https://vandebron.atlassian.net/browse/TECH-470) Rethink duplicate Jenkinsfiles and runner.py-s sam@vandebron.nl 
Currently we have many very similar Jenkinsfiles, python runners etc in three different repos:

* mpyl repo
* monorepo
* scripting

Figure out a way to centralise these, or to have safeguards to not break them with a breaking change

----

The github action to sync Jenkinsfiles back to the scripting repo could be a good starting point for keeping the repo-runners synced

[https://github.com/Vandebron/Vandebron/blob/master/.github/workflows/push*Jenkinsfile_to_scripting.yml](https://github.com/Vandebron/Vandebron/blob/master/.github/workflows/push_Jenkinsfile_to*scripting.yml) 

Things like repository link, mpyl*config path and run*properties path would need to be configurable/overwriteable through a github action

🏗️ Build [5](https://jenkins.k8s-serv-backend.vdbinfra.nl/job/MPyL%20Pipeline%20-%20Test/job/PR-194/5/display/redirect) ✅ Successful, started by _Sam Theisens_  
🚀 *[cloudfront-service](https://cloudfront-service-194.test.nl/)*, *job*, *kong-sync*, *[nodeservice](https://nodeservice-194.test.nl/)*, *[sbtservice](https://sbtservice-194.test.nl/)*, *sparkJob*  

[TECH-470]: https://vandebron.atlassian.net/browse/TECH-470?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README-dev.md [hunk 1]
```diff
@@ -50,13 +50,17 @@ IDE [here](https://black.readthedocs.io/en/stable/integrations/editors.html).
...
-gh release create 0.0.2 --generate-notes
-```
-
-which will trigger a build and release to https://pypi.org/project/mpyl/
+§§1. Check the latest [release number](https://github.com/Vandebron/mpyl/releases) 
+2. Add the new release number to [mpyl/cli/releases/releases.txt](src/mpyl/cli/releases/releases.txt)
+3. Create a new release notes file in [releases/notes/](releases/notes/) and name it `<version>.md`
+Noteworthy changes should be added to this file. Think: new cli commands, new features, breaking changes, upgrade 
+instructions, etc.
+4. Merge all PRs that you want to include in the release to `main`
```
Comment (Jorg88) [inline README-dev.md:58]: nit: this should be step 1 I would say

File: releases/README.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+# Release notes
```
Comment (Jorg88) [inline releases/README.md:1]: This file is not generated right? Why do we need both this file and the file per release?

Comment (SamTheisens) [reply releases/README.md:1]: It _is_ generated :-)

File: README-dev.md (hunk 1 above)
Comment (SamTheisens) [reply README-dev.md]: I'm suggesting to consider creating release notes for each PR. Hope this makes that a bit clearer: https://github.com/Vandebron/mpyl/pull/194/commits/63a2c7456b88f302340001961bd3f18a32c73b3f

**Review by koeves — APPROVED** [review]

File: README-dev.md (hunk 1 above)
Comment (Jorg88) [reply README-dev.md]: Ahh okay!

**Review by Jorg88 — APPROVED** [review]

**Pull request merged**